HomeTask_8.py

```diff
@@ -0,0 +1,35 @@
+"""HomeTask_8.
+
+Given the list of integers ( positive , negative, zeros)
+Find max multiplication of three values in list
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+Output: Max value = "360". Nums are: (12, 5, 6)
+
+Input: [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+Output: Max value = "2016". Nums are: (-7, 12, -24)
+"""
+
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+i = 0
+l2 = [abs(num) for num in l1]
+l2.sort(reverse=True)
+max_list = []
+max_value = 1
+while i < 3:
```

Comment:
In general looks too heavy and complicated. Try to find solution without abs() for example you can sort your list and find some patterns  and then realize how to build your new algorithm. Or try  somehow to get combinations of values and operate with them.